Code/Framework/AzCore/AzCore/Serialization/DataOverlayProviderMsgs.cpp

```diff
@@ -19,8 +19,11 @@ namespace AZ
         nodeStack.push_back(m_dataContainer);
 
         SerializeContext::EnumerateInstanceCallContext callContext(
-            AZStd::bind(&DataOverlayTarget::ElementBegin, this, &nodeStack, AZStd::placeholders::_1, AZStd::placeholders::_2, AZStd::placeholders::_3),
-            AZStd::bind(&DataOverlayTarget::ElementEnd, this, &nodeStack),
+            [this, &nodeStack](void* instance_pointer, const SerializeContext::ClassData* classData, const SerializeContext::ClassElement* classElement)->bool
+            {
+                return ElementBegin(&nodeStack,instance_pointer,classData,classElement);
+            },
+            [this,&nodeStack]()->bool { return ElementEnd(&nodeStack); },
```

Comment:
So the variable naming convention should use [lowerCamelCase](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#variable-names) for variable names.
So `instance_pointer` should be `instancePointer.

Now for some nitpicks:
The `ElementBegin` call is spaces after each comma.
Same for the second element, there is a missing space between this and &nodeStack

Reply:
Fixed in 844cc43